### DISCUSSION TITLE
what is this error?

### DISCUSSION THREAD

**Discussion by HavocShreds** [gucci-on-fleek/lockdown-browser#55] [Q&A]
![Screenshot 2024-02-13 182746](https://github.com/gucci-on-fleek/lockdown-browser/assets/159975664/60a630fd-9dbd-4ef9-b2f9-55b63f89b7c4)

This is my first time using sandbox, If anyone can help me set this up, I'd appreciate it. Currently I have followed all the steps in the "system requirement section" and the "demo video."

**Comment by gucci-on-fleek** [comment]
`cd .\path\to\directory` moves you to that directory, but you have `cd .\path\to\file .\path\to\directory` which doesn't really make any sense since (1) `cd` only takes 1 path, but you gave it 2; and (2) `LockDown-etc.exe` is a path to a file, but `cd` only works with directories.

**Reply by HavocShreds** [reply]
are you able to help me with this.

**Reply by gucci-on-fleek** [reply]
It's much easier for me to keep the discussion here.

**Reply by HavocShreds** [reply]
Can you explain the steps for powershell from step 1 to the last step. Do I need to place the browser file anywhere specific or it can stay in my download folder and I can access it from there. "cd .\path\to\file .\path\to\directory" I typed it like so before because this is how it was shown in the demo video, unless I'm mistaken.

**Reply by gucci-on-fleek** [reply]
> Can you explain the steps for powershell from step 1 to the last step.

It should be explained fairly well [on the main page](https://github.com/gucci-on-fleek/lockdown-browser?tab=readme-ov-file#system-requirements), with [a little more detail in this comment](https://github.com/gucci-on-fleek/lockdown-browser/discussions/24#discussioncomment-6839457).

But _very_ roughly, the steps are

```powershell
# Clone the repo
git clone https://github.com/gucci-on-fleek/lockdown-browser.git

# Enter the repo folder
cd lockdown-browser

# Copy the installer over. You can copy it using Explorer if you prefer.
cp C:\path\to\LockDown-etc.exe .\runtime_directory  

# Build the executable files. Make sure that you've already installed Visual C++ or this won't work.
.\build.ps1

# Run the Sandbox. You can double-click the file in Explorer if you prefer.
.\runtime_directory\Sandbox.wsb
```

> "cd .\path\to\file .\path\to\directory" I typed it like so before because this is how it was shown in the demo video, unless I'm mistaken.

Ahhhh, I see where the mixup is. It's `cp Lockdown-etc.exe .\lockdown-browser\runtime_directory` (`cp` for **C**o**P**y) followed by `cd .\lockdown-browser` (`cd` for **C**hange **D**irectory).

> Do I need to place the browser file anywhere specific or it can stay in my download folder and I can access it from there. "

Yes, the file needs to be placed inside `runtime_directory` (that's what the `cp` command does).
